packages/app/package.json

```diff
@@ -15,63 +15,58 @@
+    "@backstage/core-app-api": "^1.12.4",
+    "@backstage/core-components": "^0.14.6",
+    "@backstage/core-plugin-api": "^1.9.2",
+    "@backstage/integration-react": "^1.1.26",
+    "@backstage/plugin-api-docs": "^0.11.4",
+    "@backstage/plugin-catalog": "^1.19.0",
     "@backstage/plugin-catalog-common": "^1.0.22",
-    "@backstage/plugin-catalog-graph": "^0.4.3",
-    "@backstage/plugin-catalog-import": "^0.10.9",
-    "@backstage/plugin-catalog-react": "^1.11.2",
-    "@backstage/plugin-github-actions": "^0.6.14",
-    "@backstage/plugin-org": "^0.6.23",
-    "@backstage/plugin-permission-react": "^0.4.21",
-    "@backstage/plugin-scaffolder": "^1.19.2",
-    "@backstage/plugin-search": "^1.4.9",
-    "@backstage/plugin-search-react": "^1.7.9",
-    "@backstage/plugin-tech-radar": "^0.7.2",
-    "@backstage/plugin-techdocs": "^1.10.3",
-    "@backstage/plugin-techdocs-module-addons-contrib": "^1.1.8",
-    "@backstage/plugin-techdocs-react": "^1.2.2",
-    "@backstage/plugin-user-settings": "^0.8.4",
-    "@backstage/theme": "^0.5.2",
-    "@material-ui/core": "^4.12.4",
-    "@material-ui/icons": "^4.11.3",
-    "@openshift/dynamic-plugin-sdk": "^4.0.0",
-    "@scalprum/core": "^0.6.1",
-    "@scalprum/react-core": "^0.6.1",
-    "history": "^5.3.0",
-    "lodash": "^4.17.21",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-router": "^6.17.0",
-    "react-router-dom": "^6.17.0",
-    "react-use": "^17.4.0"
+    "@backstage/plugin-catalog-graph": "^0.4.4",
+    "@backstage/plugin-catalog-import": "^0.10.10",
+    "@backstage/plugin-catalog-react": "^1.11.3",
+    "@backstage/plugin-org": "^0.6.24",
+    "@backstage/plugin-permission-react": "^0.4.22",
+    "@backstage/plugin-scaffolder": "^1.19.3",
+    "@backstage/plugin-search": "^1.4.10",
+    "@backstage/plugin-search-react": "^1.7.10",
+    "@backstage/plugin-techdocs": "^1.10.4",
+    "@backstage/plugin-techdocs-module-addons-contrib": "^1.1.9",
+    "@backstage/plugin-techdocs-react": "^1.2.3",
+    "@backstage/plugin-user-settings": "^0.8.5",
+    "@emotion/react": "^11.11.4",
+    "@mui/icons-material": "^5.15.16",
+    "@mui/material": "^5.15.16",
+    "@mui/styles": "^5.15.16",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-router": "^6.23.0",
+    "react-router-dom": "^6.23.0",
+    "tss-react": "^4.9.10",
+    "@backstage-community/plugin-github-actions": "^0.6.16",
+    "@backstage-community/plugin-tech-radar": "^0.7.4"
   },
   "devDependencies": {
-    "@backstage/cli": "0.26.2",
-    "@backstage/test-utils": "1.5.3",
-    "@janus-idp/cli": "*",
-    "@scalprum/react-test-utils": "0.0.4",
-    "@testing-library/jest-dom": "6.4.2",
-    "@testing-library/react": "14.2.1",
-    "@testing-library/user-event": "14.5.1",
-    "@types/node": "18.18.5",
-    "@types/react-dom": "18.2.18",
-    "cross-env": "7.0.3"
+    "@backstage/cli": "0.26.4",
+    "@backstage/test-utils": "1.5.4",
+    "@playwright/test": "1.32.3",
```

Comment:
Is this added here on purpose ?
I ran into errors while executing the playwright tests
Here's my PR https://github.com/schultzp2020/backstage-plugins/pull/4

Reply:
No, this was included in the default frontend template.